src/components/base/JLogo/tests/JLogo.test.js

```diff
@@ -0,0 +1,68 @@
+// @flow strict
+
+import React from 'react'
+import { mount } from 'enzyme'
+
+jest.mock('react-router5', () => ({
+  withRouter: function withRouter(BaseComponent) {
+    const ReactForMock = require('react')
+
+    return function WithRouter(props) {
+      return ReactForMock.createElement(
+        BaseComponent, {
+          ...props,
+          router: {
+            matchUrl: () => null,
+          },
+        },
+      )
+    }
+  },
+}))
+
+// eslint-disable-next-line import/first
+import { JLogo } from '../JLogo'
+
+describe('JLogo', () => {
+  test('is available', () => {
+    expect(JLogo).toBeDefined()
+  })
+
+  test('renders', () => {
+    const wrapper = mount(<JLogo />)
+
+    expect(wrapper.children()).toHaveLength(1)
+    expect(wrapper.prop('className')).toBe('')
+    expect(wrapper.prop('theme')).toBe('white')
+
+    const jLink = wrapper.childAt(0)
+
+    expect(jLink.prop('href')).toBe('/')
+    expect(jLink.hasClass('__j-logo')).toBe(true)
+  })
+
+  test('renders (specified theme)', () => {
+    const wrapper = mount(<JLogo theme='blue' />)
+
+    expect(wrapper.prop('theme')).toBe('blue')
```

Comment:
What is tested here? If prop is specified, then it is specified. This is tested by React core members, no need to reproduce it.